commands/xtrim.md

```diff
@@ -10,10 +13,20 @@ the latest 1000 items:
 XTRIM mystream MAXLEN 1000
 ```
 
+To give another exmaple, the following command will trim the stream
+so that it won't contain entries with IDs smaller than 649085820:
+
+```
+XTRIM mystream MINID 649085820-0
+```
+
 By default, or when provided with the optional `=` argument, the command
-performs exact trimming. That means that the trimmed stream's length will be
```

Comment:
here's a good example of why starting a new sentence in an existing line causes an unnecessary diff